python/ray/tune/examples/ax_example.py

```diff
@@ -55,62 +54,32 @@ def easy_objective(config):
 
     ray.init()
 
-    config = {
+    tune_kwargs = {
         "num_samples": 10 if args.smoke_test else 50,
         "config": {
             "iterations": 100,
+            "x1": tune.uniform(0.0, 1.0),
+            "x2": tune.uniform(0.0, 1.0),
+            "x3": tune.uniform(0.0, 1.0),
+            "x4": tune.uniform(0.0, 1.0),
+            "x5": tune.uniform(0.0, 1.0),
+            "x6": tune.uniform(0.0, 1.0),
         },
         "stop": {
             "timesteps_total": 100
         }
     }
-    parameters = [
```

Comment:
wow.......................

Reply:
maybe actually you could consider leaving this in, just as a note that there is equivalence.